sessions_core/01_introduction.qmd

```diff
@@ -0,0 +1,751 @@
+:::
+
+You should see an interface that looks something like this:
+
+![View of the Rstudio IDE interface at opening](../img/core/01_introduction/01_rstudio_interface_en.png){fig-align="center"}
+
+::: {.look}
+Inspect the RStudio interface.
+:::
+
+You will have either three or four panels, including:
+
+
+#### Uper Right Corner
+
+To the upper right there will be a panel with several tabs. Many of these are beyond the scope of this course, and we will focus on these two:
+
+- *Environment.* A list of the objects{.hoverinfo toggle='tooltip' title='You will learn later this lesson that in R, pretty much everything is an object, including functions, scalar values, and other more complex data structures'} saved by the user in the current [session]{.hoverinfo toggle='tooltip' title='You can think of an R session like you would think of starting up a computer. Whenever a session starts, everything is blank and ready for computation in the same way that there aren't any programs open when you first turn on your computer. In general, we encourge you to stop and start your R sessions regularly, you may just find that _turning it off an on again_ will fix some of your bugs!'}. Because you’ve just started a new session, your environment should be empty.
+
+- *History.* A record of all the commands you have executed during the current session.
+
+#### Bottom Right Corner
+
+To the bottom right there will be another multi tab panel, including:
+
+- *Files.* A file explorer for the [working directory]{.hoverinfo toggle='tooltip' title="The working directory is the folder location (directory) where R is currently working. We will talk more about the idea of paths and working directories in the next session"}.
+- *Plots.* A location where RStudio will display *static* visualizations; this tab should be empty for the moment.
+- *Packages.* A list of all the [R packages]{.hoverinfo toggle='tooltip' title="Packages are simply collections of functions that help extend the functionality of R. We will discuss them in greater detail in the next lesson."} installed on your computer.
+- *Help.* A place to read help pages and documentation for functions and packages.[Documentation refers to additional narrative text either in a code file or an external document that is written to help folks understand what a piece of code, function, or package does and how it should be used. Code documentation is a bit analogous to creating a data dictionary for a set of survey data and is an invaluable part of making robust and reusable code]{.aside}
+- *Viewer.* A location where RStudio will display [html outputs]{.hoverinfo toggle='data' title="These could include, for example, tables, interactive widgets, or even full on dashboards."}.
+
+#### Left Side
+To the left or bottom left (if you have four panels) you should see the *console* (we will come back to it in @sec-console).
```

Comment:
i think the idea of linking sections is cool but in the rendered document they appear with the syntax "Section 3.2" etc. this is a bit confusing as the actual document doesn't use this numbering system, simillarly the parenthesis doesn't quite make sense to me. 

is this a syntax issue?